Libraries/Core/ExceptionsManager.js

```diff
@@ -12,6 +12,8 @@
 
 import type {ExtendedError} from 'parseErrorStack';
 
+const INTERNAL_CALLSITES_REGEX = /ReactNativeRenderer-dev\.js|MessageQueue\.js/;
```

Comment:
Actually should we make this an exact match with `^` and `$`?

Reply:
For now I've added the `$` but I'm going to compare how the paths look in normal app vs RNTester, because they differ.

Reply:
Adjusted the paths: eb97ee3b2a27d0de3ddb8fe4f4988ce0172f8e85. They're now generated from an array, so it should be easier to add new ones (or generalize more, we'll see about that).